airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py

```diff
@@ -106,4 +107,4 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
             return False, e
 
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
-        return [ExchangeRates(config.get("base"), config["start_date"], config["access_key"])]
+        return [ExchangeRates(config.get("base"), config["start_date"], config["access_key"], config["ignore_weekends"])]
```

Comment:
Please change this to get a default value, this will prevent backward compatibility.
```suggestion
        return [ExchangeRates(config.get("base"), config["start_date"], config["access_key"], config.get("ignore_weekends", True))]
```